OxfordBodleian/Juel-Jensen/BDLaethb5.xml

```diff
@@ -33,28 +33,8937 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                        <locus from="97ra" to="97rb"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Heda21Dibora"
+                                            >Commemoration of the <foreign xml:lang="la">translatio</foreign> of Diborā</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.21.6">
+                                        <locus target="#97rb"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda21SalMary">Salām-hymn to
+                                            Mary</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.21.7">
+                                        <locus target="#97rb"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda21SalGregory">Salām-hymn to
+                                            Gregory Thaumaturgos</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.21.8">
+                                        <locus target="#97rb"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda21SalQozmos">Salām-hymn to
+                                            Qozmos</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.21.9">
+                                        <locus target="#97rb"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda21SalYohannes">Salām-hymn to
+                                            Yoḥannǝs</title>
+                                    </msItem>
+
+                                    <msItem xml:id="ms_i1.4.21.10">
+                                        <locus from="97rb" to="97rc"/>
+                                        <title type="complete"
+                                            ref="LIT4035SenkessarDL#Heda21SalEsmnunay">Salām-hymn to
+                                            the martyrs of ʾƎsmunayn</title>
+                                    </msItem>
+                                </msItem>
+
+                                <msItem xml:id="ms_i1.4.22">
+                                    <locus from="97rc" to="97vc"/>
+                                    <title type="complete" ref="LIT4035SenkessarDL#Heda22"/>
+
+                                    <msItem xml:id="ms_i1.4.22.1">
+                                        <locus from="97rc" to="97vc"/>
+                                        <title type="complete" ref="LIT4035SenkessarDL#Heda22Qozmos"
```

Comment:
ref is correct? "Qozmos" not: Tewodyādā

Reply:
It's correct, Tewodyada is the mother, she and her children, the first of whom is Qozmos (the first name in the commemoration), are commemorated. But I deleted the "the" now ;)